src/passes/RemoveMemoryInit.cpp

```diff
@@ -15,20 +15,25 @@
  */
 
 //
-// Removeds memory segments, leaving only code in the module.
+// Removes memory segments, leaving only code in the module. It also removes
+// the start function, which is only used for initializing the memory.
```

Comment:
```suggestion
// the start function, which this pass assumes is only used for initializing
// the memory.
```

Reply:
@sbc100 said that at least lld guarantees this. So how about

// the start function, which (for LLVM use cases) is only used for initializing the memory

Reply:
sgtm, yeah, either way it's good to clarify this is for LLVM output.